docs/standard/asynchronous-programming-patterns/asynchronous-programming-model-apm.md

```diff
@@ -15,31 +15,31 @@ author: "rpetrusha"
 ms.author: "ronpet"
 ---
 # Asynchronous Programming Model (APM)
-An asynchronous operation that uses the <xref:System.IAsyncResult> design pattern is implemented as two methods named **Begin***OperationName* and **End***OperationName* that begin and end the asynchronous operation *OperationName* respectively. For example, the <xref:System.IO.FileStream> class provides the <xref:System.IO.FileStream.BeginRead%2A> and <xref:System.IO.FileStream.EndRead%2A> methods to asynchronously read bytes from a file. These methods implement the asynchronous version of the <xref:System.IO.FileStream.Read%2A> method.  
+An asynchronous operation that uses the <xref:System.IAsyncResult> design pattern is implemented as two methods named **Begin**_OperationName_ and **End**_OperationName_ that begin and end the asynchronous operation *OperationName* respectively. For example, the <xref:System.IO.FileStream> class provides the <xref:System.IO.FileStream.BeginRead%2A> and <xref:System.IO.FileStream.EndRead%2A> methods to asynchronously read bytes from a file. These methods implement the asynchronous version of the <xref:System.IO.FileStream.Read%2A> method.  
```

Comment:
>  **Begin**_OperationName_

That looks strange to me. It is the bold contrasted with the emphasis when members are usually rendered as `code`. What about

> `BeginOperationName`

It is fine if we want to take the PR as-is because I see it is a mechanical fix to a bad markdown translation. If we do that, then we should file an issue.

Reply:
I'd rather fix these in this PR.  I think these issues were added in the previous automated markdown translation.

I agree with @richlander 's recommendation.